src/nvim/eval.h

```diff
@@ -3,6 +3,47 @@
 
 #include "nvim/profile.h"
 
+// All user-defined functions are found in this hashtable.
+hashtab_T func_hashtab;
```

Comment:
Not sure if this is something we're changing, but the current convention I see is to use `EXTERN` (defined in `nvim/globals.h`), which initializes it when included from `main.c`.